package.json

```diff
@@ -3,47 +3,20 @@
   "version": "1.0.0",
   "scripts": {
     "build": "rollup -c",
-    "lint": "eslint lib/ test/",
+    "lint": "standard",
```

Comment:
We don't need a separate script for this anymore. Let's inline it into the `test` script.